doc/src/site/sphinx/ml/sw_drf.rst

```diff
@@ -31,17 +39,33 @@ and :ref:`model_details_H2ODRFMOJOModel`.
         .. code:: scala
 
             import org.apache.spark.SparkFiles
-            spark.sparkContext.addFile("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/prostate/prostate.csv")
-            val rawSparkDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("prostate.csv"))
-            val sparkDF = rawSparkDF.withColumn("CAPSULE", $"CAPSULE" cast "string")
+            val datasetUrl = "https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/cars_20mpg.csv"
+            spark.sparkContext.addFile(datasetUrl) //for example purposes, on a real cluster it's better to load directly from distributed storage
+            val sparkDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("prostate.csv"))
```

Comment:
I think this line will fail after  you changed dataset on the [line 42](https://github.com/h2oai/sparkling-water/pull/2771/files#diff-42ff28ef2817f3ca584bc10683a36681b3d4c5195ce4b01e653cdbb772806ab8R42).